guides/manual-installation.md

```diff
@@ -247,6 +247,7 @@ AMQP_URL=amqp://trento_user:trento_user_password@localhost:5672/vhost
 DATABASE_URL=ecto://trento_user:web_password@localhost/trento
 EVENTSTORE_URL=ecto://trento_user:web_password@localhost/trento_event_store
 ENABLE_ALERTING=false
+ENABLE_OIDC=false
```

Comment:
You can document these variables, but they are not mandatory

Reply:
I removed this lines,  like you are saying, they are not mandatory, we explain it explicitly in the integration section. 
Additionally this would require to update the official documentation too, better to keep it spitted!